db/migrate/20191101113230_add_revoked_at_to_api_key.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+class AddRevokedAtToApiKey < ActiveRecord::Migration[5.2]
+  def change
+    add_column :api_keys, :revoked_at, :datetime
+    add_column :api_keys, :description, :text
+
+    execute "INSERT INTO site_settings(name, data_type, value, created_at, updated_at)
```

Comment:
technically I am not sure this is correct given this is a reversable migration, maybe just define the up and throw an exception on down?